### PR TITLE
Complete python loops.

### PR DESCRIPTION
**Pull request by kevinmbeck**
Screen Shot 2022-04-25 at 8 45 27 PM

### CODE REVIEW

File: ds-prep-python-loops/main.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+record = ("1", "Grimdiana", "Bones", "boulders")
+row = " "
```
Comment (TimelikeClosure) [inline ds-prep-python-loops/main.py]: This isn't an empty string, as it has a space (`" "`) in it:

---

image

File: ds-prep-python-loops/main.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+for x in record:
+  row = (row + x + ",")
+  print(row)
```
Comment (TimelikeClosure) [inline ds-prep-python-loops/main.py]: The value of `row` should only be printed once, at the end:

---

image

File: ds-prep-python-loops/main.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "Today": "C",
+    "I Don't Care": "SS"
+}
+players = [ ]
+for x in player_positions:
+  players.append(x)
+print(players)
+positions = set()
+for x in player_positions:
+  positions.add(player_positions[x])
```
Comment (TimelikeClosure) [inline ds-prep-python-loops/main.py]: This should directly loop through the values, not loop through the keys and generate the values from them:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: ds-prep-python-loops/main.py (hunk 2 above)
Comment (kevinmbeck) [reply ds-prep-python-loops/main.py]: But doesn't it say in the step before that to concatenate row, the value and a comma as a string and reassign all of that back to row

**Comment by kevinmbeck** [comment]
I fixed the row issue. I misread your comment about row. This is how it prints now 
Screen Shot 2022-04-26 at 2 45 48 PM

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**